hasjob/static/js/app.js

```diff
@@ -216,26 +194,46 @@ window.Hasjob.Filters = {
       selectedCurrency: window.Hasjob.Config.selectedFilters.currency,
       pay: window.Hasjob.Config.selectedFilters.pay,
       equity: window.Hasjob.Config.selectedFilters.equity,
-      sidebarOn: false
+      isMobile: $(window).width() < 768
```

Comment:
Is 768 a safe number to hardcode?

Reply:
Our current layout, screen size >= 768px the filters menu is not a dropdown and is a horizontal bar below the header